src/db/models/User.ts

```diff
@@ -105,9 +105,8 @@ UserModel.init(
         },
         commercial_use_reason: {
             type: DataTypes.STRING,
-            validate: {
-                is: NAME_REGEX,
-            },
+            allowNull: true,
+            validate: { validate: (value: string) => (value ? NAME_REGEX.test(value) : true) },
```

Comment:
je fais ca car NAME_REGEX fail avec la value vide `''`

Reply:
`value === '' || NAME_REGEX.test(value)`

so that we avoid falsy values such as undefined or similar.